nuxt/components/edit/EditDraftLyrics.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    persistent
+    fullscreen
+    no-click-animation
+    hide-overlay
+  >
+    <template #activator="{ on }">
+      <v-btn
+        icon
+        v-on="on"
+      >
+        <v-icon>lyrics</v-icon>
+      </v-btn>
+    </template>
+    <v-card :loading="loading">
+      <v-app-bar
+        fixed
+        elevate-on-scroll
+      >
+        <v-btn
+          icon
+          @click="close"
+        >
+          <v-icon>close</v-icon>
+        </v-btn>
+        <v-toolbar-title>Draft Lyrics for {{ track.title }}</v-toolbar-title>
+        <v-spacer />
+        <div class="toolbar__actions">
+          <v-btn
+            v-if="isModerator && draftLyrics"
+            text
+            @click="publish"
+          >
+            Publish
+          </v-btn>
+          <v-btn
```

Comment:
I don't think I want to allow deleting draft lyrics for everyone - maybe just moderators. What do you think?

Reply:
Hmm, I was thinking it should be either moderator, or the person who last edited, however if we allow that then instead of reverting just their change they can edit everything
For now lets keep to moderator only but we should allow some way of them deleting (reverting their change or if its first then delete)

Reply:
Made the changes to keep for moderator only for now. Once you confirm will mark this as resolved